aws-athena/views/qc-vw_incorrect_asmt_value.sql

```diff
@@ -0,0 +1,59 @@
+SELECT
+    pardat.taxyr,
+    legdat.user1 AS township_code,
+    pardat.parid,
+    pardat.class,
+    pardat.luc,
+    owndat.own1,
+    aprval.reascd AS reason_for_change,
+    aprval.who,
```

Comment:
**[Question, non-blocking]** Similarly, these columns do not seem to be used for testing; are they useful for debugging failures?

Reply:
Likewise just replicating the Inquire query. I figure if we ever get to a point where we need to export these failures for consumption by analysts they'll be useful to have.